<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SketchPaper</class>
 <widget class="QWidget" name="SketchPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,1,1,2,5">
   <item row="5" column="4">
    <widget class="QSlider" name="slider_blue">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBox_green">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>두께</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>색상</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSlider" name="slider_red">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="spinBox_blue">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QSlider" name="slider_green">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>B</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>G</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="spinBox_red">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>R</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="3">
    <widget class="QLabel" name="label_color">
     <property name="styleSheet">
      <string notr="true">background-color: black</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QSpinBox" name="spinBox_thickness">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QSlider" name="slider_thickness">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_thickness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_thickness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_thickness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_thickness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
